.github/scripts/trymerge.py

```diff
@@ -619,6 +620,7 @@ def can_skip_internal_checks(pr: "GitHubPR", comment_id: Optional[int] = None) -
         return False
     return comment.author_login == "facebook-github-bot"
 
+
```

Comment:
nit: extra newline

Reply:
Err, isn't this a convention? Two blank lines between top level functions?
https://peps.python.org/pep-0008/#blank-lines

Reply:
I suppose it's an official convention, but I haven't noticed it being done to often in our repo.

Looking at this file, maybe a bit under half of the top level functions are separated this way...eh, whatevs. This is where lint-consistently-on-save tools are handy :P